Cargo.toml

```diff
@@ -6,3 +6,4 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+anyhow = "1.0"
```

Comment:
If you do want to pull in `anyhow`, the pattern [they suggest](https://docs.rs/anyhow/latest/anyhow/) is to use `Result<T, anyhow::Error`

Reply:
Done!